content/en/blog/_posts/2018-12-05-kubeadm-ga-release.md

```diff
@@ -0,0 +1,97 @@
+ * **Upgrades between minor versions** --- The [`kubeadm upgrade`](https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-upgrade/) command is now fully GA. It handles control plane upgrades for you, which includes upgrades to [etcd](https://etcd.io), the [API Server](https://kubernetes.io/docs/reference/using-api/api-overview/), the [Controller Manager](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/), and the [Scheduler](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-scheduler/). You can seamlessly upgrade your cluster between minor or patch versions (e.g. v1.12.2 -> v1.13.1 or v1.13.1 -> v1.13.3).
+ * **etcd setup** --- [etcd](https://etcd.io) is now set up in a way that is secure by default, with TLS communication everywhere, and allows for expanding to a highly available cluster when needed.
+
+### Who will benefit from a stable kubeadm
+
+SIG Cluster Lifecycle has identified a handful of likely kubeadm user profiles, although we expect that kubeadm at GA can satisfy many other scenarios as well.
+
+Here's our list:
+
+- You're a **new user** who wants to take Kubernetes for a spin. kubeadm is the fastest way to get up and running on [Linux machines](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/). If you're using [Minikube](https://github.com/kubernetes/minikube) on a Mac or Windows workstation, you're actually already running kubeadm inside the Minikube VM!
+- You're a **system administrator** responsible for setting up Kubernetes on bare metal machines and you want to quickly create Kubernetes clusters that are secure and in conformance with best practices but also highly configurable.
+- You're a **cloud provider** who wants to add a Kubernetes offering to your suite of cloud services. kubeadm is the go-to tool for creating clusters at a low level.
+- You're an **organization with special needs**. Existing public cloud offerings like [Amazon EKS](https://aws.amazon.com/eks/) and [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/) won't cut it for you; you need customized Kubernetes clusters tailored to your hardware, security, policy, and other needs.
+- You're creating a **higher-level cluster creation tool** than kubeadm, building the cluster experience from the ground up, but you don't want to reinvent the wheel. You can "rebase" on top of kubeadm and utilize the common bootstrapping tools kubeadm provides for you. Several community tools have adopted kubeadm, and it's a perfect match for [Cluster API](https://github.com/kubernetes-sigs/cluster-api) implementations.
+
+All these users can benefit from kubeadm graduating to a stable GA state.
+
+### kubeadmn survey
+
+SIG Cluster Lifecycle has put together a survey for kubeadm. It's available at [https://bit.ly/2FPfRiZ](https://bit.ly/2FPfRiZ). The purpose of the survey is to collect community feedback about kubeadm for the sake of future improvement.
```

Comment:
What about trying to do better marketing for the survey: e.g.
"Although kubeadm is GA, the SIG Cluster Lifecycle commitment to improving the user experience in managing Kubernetes clusters will continue. We are launching a survey is to collect community feedback about kubeadm for the sake of future improvement."

Reply:
Also a typo in the header here: `kubeadmn`